### DISCUSSION TITLE
What's the intent behind experiment/?

### DISCUSSION THREAD

**Discussion by Yevgnen** [ashleve/lightning-hydra-template#215] [Q&A] [answered]
Hi, thanks for the great repo!

I'm quite new to `hydra` and `lightning`. I have a question about the intent for the `experiment/` directory. How is it different from individual runs? Especially in `example_full.yaml`, when and why should we override all the settings there? 

Thanks.

**Comment by ashleve** [comment, marked as answer]
HI @Yevgnen,

The main intent behind experiment configs is to allow for version control of specific configurations. E.g. I often create a separate experiment for each combination of model and datamodule (so I can track which hyperparameters are the best for given combination).

The `example` is just to show the possibilities - it's up to you how to manage your configs.
